iceberg/iceberg-handler/src/test/results/positive/llap/iceberg_major_compaction_query_metadata.q.out

```diff
@@ -106,7 +106,7 @@ Table Parameters:
 #### A masked pattern was here ####
 	rawDataSize         	0                   
 	serialization.format	1                   
-	snapshot-count      	4                   
+	snapshot-count      	3                   
```

Comment:
-1 because now we create 1 commit, right?

Reply:
Yes

Reply:
👍